### PR TITLE
Fixed Routing For Cases and Case View * Co-Author: chavionjackson

### PR DESCRIPTION
**Pull request by ShariqAli-Dev**
# Description
 - [Loom Video](https://www.loom.com/share/a84b6670b1184f2d96f3021565189f30)
 - /cases route now renders the family list table
 - when viewing  case you are now brought to /cases/:caseID instead of /case-view

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B
- [x] Not Necessary

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by ShariqAli-Dev** [comment]
[BUG: change the route of family-list to /cases.](https://trello.com/c/pyRWsYQ1/41-bug-change-the-route-of-family-list-to-cases)

File: src/components/common/Navigation.js [hunk 1]
```diff
@@ -4,17 +4,36 @@ import { Menu } from 'antd';
...
-            <Menu.Item key="resources"><Link to="/resources" />Staff Resources</Menu.Item>
-            <Menu.Item key="services"><Link to="/services" />Services</Menu.Item>
-        </Menu>
-    );
-};
+  const [current, setCurrent] = useState(null);
 
-export default Navigation;
\ No newline at end of file
+  return (
```
Comment (stkirk) [inline src/components/common/Navigation.js:9]: were the changes to this block of code just the prettier formatter?

File: src/components/common/FamilylistTable/FamilyListTable.js [hunk 2]
```diff
@@ -53,7 +52,7 @@ const FamilyListTable = ({ family }) => {
         />
         <Column
           key="view_case"
-          render={text => <a href="/case-view">View Case</a>}
+          render={text => <a href={`/cases/:caseID`}>View Case</a>}
```
Comment (stkirk) [inline src/components/common/FamilylistTable/FamilyListTable.js:55]: this route name makes much more sense, will be easier to integrate with backend

File: src/index.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ import { ExampleDataViz } from './components/pages/ExampleDataViz';
 import { config } from './utils/oktaConfig';
 import { FamilyListTable } from './components/common/FamilylistTable';
 import { CaseViewPage } from './components/pages/CaseView';
-import {FamilyIntake} from './components/common/FamilyIntake'; 
+import { FamilyIntake } from './components/common/FamilyIntake';
```
Comment (stkirk) [inline src/index.js:23]: are we waiting to change the name of components from "Family" to "Client" when they are being worked on?

File: src/components/common/Navigation.js (hunk 1 above)
Comment (ShariqAli-Dev) [reply src/components/common/Navigation.js:9]: Yeah, if this causing conflicts in the future, we may need to add a `.prettierrc` file to standardize our formatting.

File: src/components/common/FamilylistTable/FamilyListTable.js (hunk 2 above)
Comment (ShariqAli-Dev) [reply src/components/common/FamilylistTable/FamilyListTable.js:55]: Thank you, I also plan on renaming the components for consistently within the backend in our next pull request

File: src/index.js (hunk 3 above)
Comment (ShariqAli-Dev) [reply src/index.js:23]: Yes, we are indeed waiting on these components to be renamed. I'm currently working on this right now, and it'll be shown in my next pull request

File: src/components/common/FamilylistTable/FamilyListTable.js (hunk 2 above)
Comment (wild-kard) [reply src/components/common/FamilylistTable/FamilyListTable.js:55]: I agree with this route change. Much clearer way to do it.

**Review by stkirk — APPROVED** [review]

**Review by ashtilawat23 — APPROVED** [review]

**Pull request merged**